@here/harp-mapview/lib/PickingRaycaster.ts

```diff
@@ -5,21 +5,21 @@
  */
 
 import * as THREE from "three";
-import { MapView } from "./MapView";
 
 /**
  * Raycasting points is not supported as necessary in Three.js. This class extends a
- * [[THREE.Raycaster]] except that it holds a reference to [[MapView]] allowing to access the
- * camera and the renderer from [[MapViewPoints]], in order to project the points in screen space.
+ * [[THREE.Raycaster]] by holding a reference to [[Camera]] allowing to project the points in screen
+ * space, used for example in the [[MapViewPoints]].
  */
 export class PickingRaycaster extends THREE.Raycaster {
     /**
-     * Constructs a `MapViewRaycaster`. It keeps a reference to [[MapView]] in order to access the
-     * active camera when calling the custom method `raycastPoints`.
+     * Constructor.
      *
-     * @param m_mapView the active [[MapView]].
+     * @param camera the camera used for projection to screen.
```

Comment:
Maybe you can explain a bit more that this camera is the world camera (i.e. equal to `MapView.camera`) whereas the `Raycaster.camera` is our internal RTE camera (i.e. equal to `MapView.m_rteCamera`).
Maybe it also makes sense to rename camera to worldCamera b/c otherwise we have two properties with the same name `camera` (I wonder that typescript is not complaining abut this)

Reply:
I have fixed the PickingRaycaster to take the rteCamera.